### PR TITLE
docs: fix feature list

### PR DESCRIPTION
**Pull request by frondeus**

### CODE REVIEW

File: docs/features/index.md [hunk 1]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-00005-00 | Query Service: Postgres                  | Ready      | 0.0.1       | N/A                                   |
+| CDLF-00006-00 | Automatic Query Destination              | Ready      | 0.0.1       | N/A                                   |
+| CDLF-00007-00 | System Metrics Support                   | Ready      | 0.0.1       | N/A                                   |
+| CDLF-00008-00 | Victoria Metrics Support                 | Ready      | 0.0.9       | N/A                                   |
+| CDLF-00009-00 | CDL Input Message Batching               | RC         | 0.3.1-rc    | N/A                                   |
+| CDLF-0000A-00 | Message Ordering                         | RC         | 0.0.9       | [DOC](./ordering.md)                  |
+| CDLF-0000B-00 | Access Groups                            | Suggestion | -----       | N/A                                   |
+| CDLF-0000C-00 | Full GRPC communication support          | PoC        | 0.3.1-rc    | [RFC](../rfc/CDLF-0000C-00-rfc-01.md) |
+| CDLF-0000D-00 | Service Mesh (istio)                     | Abandoned  | -----       | N/A                                   |
```
Comment (Kononnable) [inline docs/features/index.md:32]:
```suggestion
| CDLF-0000D-00 | Service Mesh (istio)                     | Retired    | -----       | N/A                                   |
```
No such status as abandoned.

**Review by esavier — APPROVED** [review]

File: docs/features/index.md (hunk 1 above)
Comment (esavier) [reply docs/features/index.md:32]: retired means it was there at some point? you released it somehow?

File: docs/features/index.md [hunk 2]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-0000E-00 | CIM MessagePack Format                   | TechSpec   | -----       | [RFC](../rfc/CDLF-0000E-00-rfc-01.md) |
```
Comment (Kononnable) [inline docs/features/index.md:33]:
```suggestion
| CDLF-0000E-00 | CDL Input Message Format - MessagePack   | TechSpec   | -----       | [RFC](../rfc/CDLF-0000E-00-rfc-01.md) |
```
No confusing abbreviation - like in CDLF-00009-00

File: docs/features/index.md [hunk 3]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-0000F-00 | Configuration Service                    | TechSpec   | -----       |                                       |
+| CDLF-00010-00 | Protocol Versioning                      | TechSpec   | -----       | [RFC](../rfc/CDLF-00010-00-rfc-01.md) |
+| CDLF-00011-00 | Basic Materialization                    | TechSpec   | 0.3.1-rc    | [RFC](../rfc/CDLF-00011-00-rfc-01.md) |
+| CDLF-00012-00 | Edge registry                            | RC         | 0.3.1-rc    | [RFC](../rfc/CDLF-00012-00-rfc-01.md) |
```
Comment (Kononnable) [inline docs/features/index.md:37]:
```suggestion
| CDLF-00012-00 | Edge registry                            | Ready      | 0.3.1-rc    | [RFC](../rfc/CDLF-00012-00-rfc-01.md) |
```

File: docs/features/index.md [hunk 4]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-00009-00 | CDL Input Message Batching               | RC         | 0.3.1-rc    | N/A                                   |
+| CDLF-0000A-00 | Message Ordering                         | RC         | 0.0.9       | [DOC](./ordering.md)                  |
+| CDLF-0000B-00 | Access Groups                            | Suggestion | -----       | N/A                                   |
+| CDLF-0000C-00 | Full GRPC communication support          | PoC        | 0.3.1-rc    | [RFC](../rfc/CDLF-0000C-00-rfc-01.md) |
+| CDLF-0000D-00 | Service Mesh (istio)                     | Abandoned  | -----       | N/A                                   |
+| CDLF-0000E-00 | CIM MessagePack Format                   | TechSpec   | -----       | [RFC](../rfc/CDLF-0000E-00-rfc-01.md) |
+| CDLF-0000F-00 | Configuration Service                    | TechSpec   | -----       |                                       |
+| CDLF-00010-00 | Protocol Versioning                      | TechSpec   | -----       | [RFC](../rfc/CDLF-00010-00-rfc-01.md) |
+| CDLF-00011-00 | Basic Materialization                    | TechSpec   | 0.3.1-rc    | [RFC](../rfc/CDLF-00011-00-rfc-01.md) |
```
Comment (Kononnable) [inline docs/features/index.md:36]:
```suggestion
| CDLF-00011-00 | Basic Materialization                    | Retired    | 0.3.1-rc    | [RFC](../rfc/CDLF-00011-00-rfc-01.md) |
```

File: docs/features/index.md (hunk 3 above)
Comment (esavier) [reply docs/features/index.md:37]: RC was released with release candidate build,
Ready means it was released and is ready to use, which RC is not (preview build)
Also if you want to change it to Ready, change version to 1.0.0

File: docs/features/index.md [hunk 5]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-00012-00 | Edge registry                            | RC         | 0.3.1-rc    | [RFC](../rfc/CDLF-00012-00-rfc-01.md) |
+| CDLF-00013-00 | Materialized views                       | RC         | 0.3.1-rc    | [RFC](../rfc/CDLF-00013-00-rfc-01.md) |
+| CDLF-00014-00 | Materialization - Filters                | Ready      | 1.0.0       | missing                               |
+| CDLF-00015-00 | CDL publishing deployment configurations | TechSpec   | -----       | [RFC](../rfc/CDLF-00015-00-rfc-01.md) |
```
Comment (Kononnable) [inline docs/features/index.md:40]: RFC, but not a feature. I have no idea if this should go there. If so what's the status?

File: docs/features/index.md [hunk 6]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-00016-00 | Schema-Registry-less CDL deployment      | Ready      | 1.0.0       | [RFC](../rfc/CDLF-00016-00-rfc-01.md) |
+| CDLF-00017-00 | Materialization - Computation            | TechSpec   | -----       |                                       |
```
Comment (Kononnable) [inline docs/features/index.md:42]: @frondeus Correct me if I'm wrong, but I think this was merged and works on simple(no joins) views.

File: docs/features/index.md [hunk 7]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-0000B-00 | Access Groups                            | Suggestion | -----       | N/A                                   |
+| CDLF-0000C-00 | Full GRPC communication support          | PoC        | 0.3.1-rc    | [RFC](../rfc/CDLF-0000C-00-rfc-01.md) |
+| CDLF-0000D-00 | Service Mesh (istio)                     | Abandoned  | -----       | N/A                                   |
+| CDLF-0000E-00 | CIM MessagePack Format                   | TechSpec   | -----       | [RFC](../rfc/CDLF-0000E-00-rfc-01.md) |
+| CDLF-0000F-00 | Configuration Service                    | TechSpec   | -----       |                                       |
+| CDLF-00010-00 | Protocol Versioning                      | TechSpec   | -----       | [RFC](../rfc/CDLF-00010-00-rfc-01.md) |
+| CDLF-00011-00 | Basic Materialization                    | TechSpec   | 0.3.1-rc    | [RFC](../rfc/CDLF-00011-00-rfc-01.md) |
+| CDLF-00012-00 | Edge registry                            | RC         | 0.3.1-rc    | [RFC](../rfc/CDLF-00012-00-rfc-01.md) |
+| CDLF-00013-00 | Materialized views                       | RC         | 0.3.1-rc    | [RFC](../rfc/CDLF-00013-00-rfc-01.md) |
```
Comment (Kononnable) [inline docs/features/index.md:38]:
```suggestion
| CDLF-00013-00 | Materialized views                       | Ready      | 0.3.1-rc    | [RFC](../rfc/CDLF-00013-00-rfc-01.md) |
```

File: docs/features/index.md [hunk 8]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-00014-00 | Materialization - Filters                | Ready      | 1.0.0       | missing                               |
+| CDLF-00015-00 | CDL publishing deployment configurations | TechSpec   | -----       | [RFC](../rfc/CDLF-00015-00-rfc-01.md) |
+| CDLF-00016-00 | Schema-Registry-less CDL deployment      | Ready      | 1.0.0       | [RFC](../rfc/CDLF-00016-00-rfc-01.md) |
+| CDLF-00017-00 | Materialization - Computation            | TechSpec   | -----       |                                       |
+| CDLF-00018-00 | Materialization - Materialized Types     | TechSpec   | -----       |                                       |
+| CDLF-00019-00 | Materializer - OnDemand                  | Ready      | 1.0.0       | missing                               |
+| CDLF-0001A-00 | Materializer - General                   | Ready      | 1.0.0       | missing                               |
```
Comment (Kononnable) [inline docs/features/index.md:45]:
```suggestion
```
Not a feature by itself - covered in CDLF-00013-00

File: docs/features/index.md [hunk 9]
```diff
@@ -15,36 +15,38 @@
...
+| CDLF-0001B-00 | Materialization - Notifications          | Ready      | 1.0.0       | missing                               |
+| CDLF-0001C-00 | Object-side configuration                | Suggestion | -----       |                                       |
+| CDLF-0001D-00 | CIM Object Valdiation                    | Suggestion | -----       |                                       |
+| CDLF-0001E-00 | Materialization - Relationships          | Idea       | -----       |                                       |
```
Comment (Kononnable) [inline docs/features/index.md:49]:
```suggestion
| CDLF-0001E-00 | Materialization - Relationships          | Suggestion | -----       |                                       |
```
No idea type

File: docs/features/index.md (hunk 5 above)
Comment (esavier) [reply docs/features/index.md:40]: yes, its not a feature, it should go in form of rfc but to documentation folder probably

File: docs/features/index.md (hunk 7 above)
Comment (esavier) [reply docs/features/index.md:38]: if ready then change version to 1.0.0

File: docs/features/index.md (hunk 8 above)
Comment (esavier) [reply docs/features/index.md:45]: new component is a feature

File: docs/features/index.md (hunk 1 above)
Comment (Kononnable) [reply docs/features/index.md:32]: Abandoned is not defined anywhere, so we should either mark it as state feature may be in, or change a state to some existing state.

Comment (esavier) [reply docs/features/index.md:32]: okay, i am fine with that

File: docs/features/index.md (hunk 9 above)
Comment (Kononnable) [reply docs/features/index.md:48]: Missed that, thanks

**Review by esavier — APPROVED** [review]

File: docs/features/index.md [hunk 10]
```diff
@@ -7,44 +7,45 @@
...
+| CDLF-00009-00 | CDL Input Message Batching               | Ready      | 1.0.0       | N/A                                   |
+| CDLF-0000A-00 | Message Ordering                         | Ready      | 1.0.0       | [DOC](./ordering.md)                  |
+| CDLF-0000B-00 | Access Groups                            | Suggestion | -----       | N/A                                   |
+| CDLF-0000C-00 | Full GRPC communication support          | Ready      | 1.0.0       | [RFC](../rfc/CDLF-0000C-00-rfc-01.md) |
+| CDLF-0000D-00 | Service Mesh (istio)                     | Abandoned  | -----       | N/A                                   |
+| CDLF-0000E-00 | CDL Input Message Format - MessagePack   | TechSpec   | -----       | [RFC](../rfc/CDLF-0000E-00-rfc-01.md) |
+| CDLF-0000F-00 | Configuration Service                    | TechSpec   | -----       |                                       |
+| CDLF-00010-00 | Protocol Versioning                      | TechSpec   | -----       | [RFC](../rfc/CDLF-00010-00-rfc-01.md) |
+| CDLF-00011-00 | Basic Materialization                    | Retired    | -----       | [RFC](../rfc/CDLF-00011-00-rfc-01.md) |
```
Comment (Kononnable) [inline docs/features/index.md:36]:
```suggestion
| CDLF-00011-00 | Basic Materialization                    | Abandoned  | -----       | [RFC](../rfc/CDLF-00011-00-rfc-01.md) |
```
Materialization was never developed using this proposal.

File: docs/features/index.md [hunk 11]
```diff
@@ -7,44 +7,45 @@
...
+| CDLF-00007-00 | System Metrics Support                   | Ready      | 0.0.1       | N/A                                   |
+| CDLF-00008-00 | Victoria Metrics Support                 | Ready      | 0.0.9       | N/A                                   |
+| CDLF-00009-00 | CDL Input Message Batching               | Ready      | 1.0.0       | N/A                                   |
+| CDLF-0000A-00 | Message Ordering                         | Ready      | 1.0.0       | [DOC](./ordering.md)                  |
+| CDLF-0000B-00 | Access Groups                            | Suggestion | -----       | N/A                                   |
+| CDLF-0000C-00 | Full GRPC communication support          | Ready      | 1.0.0       | [RFC](../rfc/CDLF-0000C-00-rfc-01.md) |
+| CDLF-0000D-00 | Service Mesh (istio)                     | Abandoned  | -----       | N/A                                   |
+| CDLF-0000E-00 | CDL Input Message Format - MessagePack   | TechSpec   | -----       | [RFC](../rfc/CDLF-0000E-00-rfc-01.md) |
+| CDLF-0000F-00 | Configuration Service                    | TechSpec   | -----       |                                       |
```
Comment (Kononnable) [inline docs/features/index.md:34]:
```suggestion
| CDLF-0000F-00 | Configuration Service                    | Suggestion | -----       |                                       |
```
No document or even written requirements exist as far as I recall.

**Pull request merged**